src/routes/apiConfig.js

```diff
@@ -32,7 +32,9 @@ module.exports = async (fastify, options, next) => {
 			},
 			maxDistance: fastify.config.tracking.maxDistance,
 			everythingFlagPermissions: fastify.config.tracking.everythingFlagPermissions,
-
+			disabledHooks: ["Pokemon", "Raid", "Pokestop", "Invasion", "Lure", "Quest", "Weather", "Nest", "Gym"].flatMap((hookType) => (
```

Comment:
I had to bend my head around this expression.

Does this achieve the same thing in a simpler form?

`["Pokemon", "Raid", "Pokestop", "Invasion", "Lure", "Quest", "Weather", "Nest", "Gym"].filter((hookType) => this.config.general[`disable${hookType}`]).map((hookType) => hookType.toLowerCase())`